TestCases/parallel_regression.py

```diff
@@ -1329,7 +1329,31 @@ def main():
     species2_primitiveVenturi_mixingmodel_viscosity.test_vals = [-5.157644, -4.362599, -4.283856, -5.463200, 0.081487, -5.318960, 5.000000, -2.004278, 5.000000, -5.185927, 5.000000, -1.245596, 2.408977, 0.958168, 0.605425, 0.845383]
     species2_primitiveVenturi_mixingmodel_viscosity.new_output = True
     test_list.append(species2_primitiveVenturi_mixingmodel_viscosity)
-
+    
+    # 2 species (1 eq) primitive venturi mixing using mixing model including heat capacity and mass diffusivity
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2           = TestCase('species2_primitiveVenturi_mixingmodel_heatcapacity_H2.cfg')
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.cfg_dir   = "species_transport/venturi_primitive_3species"
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.cfg_file  = "species2_primitiveVenturi_mixingmodel_heatcapacity_H2.cfg"
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.test_iter = 50
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.test_vals = [-6.205645, -5.061525, -4.904204, -7.393911, 2.744084, -5.738068, 30.000000, -6.781415, 7.000000, -8.011696, 10.000000, -8.265141, 2.089975, 1.000000, 0.600000, 0.489975]
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.su2_exec  = "mpirun -n 2 SU2_CFD"
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.timeout   = 1600
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.new_output = True
+    species2_primitiveVenturi_mixingmodel_heatcapacity_H2.tol       = 0.00001
+    test_list.append(species2_primitiveVenturi_mixingmodel_heatcapacity_H2)
+    
+    # 2 species (1 eq) primitive venturi mixing using mixing model including heat capacity and mass diffusivity NonDimensional case
```

Comment:
Converges to the same result as the case above? If so it would be good for future debugging to comment and indicate which of the test_vals should be the same at convergence.

Reply:
Hi, thanks for the comment. Could you clarify a bit more? I think you mean that if the test case called species2_primitiveVenturi_mixingmodel_heatcapacity_H2 is the same as the previous test case. Both are different because the H2 one test the unity lewis assumption and the different heat capacities. But I might be mistaken about your comment :). Thank you so much in advance.

Reply:
The comment is that it would be good for the regressions to check equivalent dimensional and non-dimensional cases, that way there is a simple way in the future to detect a regression, otherwise we may disregard a small change in the ND case, thinking it is numerical noise, when it could be a discrepancy between dimensional and non-dimensional.
It's a nice thing to have that's all.

Reply:
Got it, Thank you so much for the clarification!! I will add the respective non-dimensional case. Thanks again!